### PR TITLE
Added a method to write a name to qrcode [#29]

### LINKED ISSUE #29: Write employee's name and equimpment id on QR code
**Is your feature request related to a problem? Please describe.**
It will be convenient to have employee's first/last name and equipment id written n QR code image (sticker). This will help to quickly identify who owns some equipment without looking into db.

**Describe the solution you'd like**
Method that writes the required info on QR code PNG file.

**Additional context**
See [ImageDraw module](https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html) for the reference

### PR DESCRIPTION
**Pull request by MykytaKyt**
# What does this PR do?

Added method that writes the full name on QR code PNG file.

## Issue reference

Fixes #29

## Explanation

Added the ability to write a name in qr code. The only disadvantage is that it is necessary to store the file with the font separately

### CODE REVIEW

File: examples/qr-generator-demo/qr-generator-demo.py [hunk 1]
```diff
@@ -12,7 +14,10 @@ def image_generator(csv_file_path):
         qr.add_data(df_list)
         qr.make()
 
+        font = ImageFont.truetype("arial.ttf", 20)
```
Comment (aartiukh) [inline examples/qr-generator-demo/qr-generator-demo.py]: we can use a default font, see `ImageFont.load_default() `

File: examples/qr-generator-demo/qr-generator-demo.py [hunk 2]
```diff
@@ -12,7 +14,10 @@ def image_generator(csv_file_path):
...
         img = qr.make_image()
+        draw = ImageDraw.Draw(img)
+        draw.text((140, 10), df["employee_name"][i], font=font)
```
Comment (aartiukh) [inline examples/qr-generator-demo/qr-generator-demo.py]: Create variables for all magic numbers

**Review by aartiukh — CHANGES_REQUESTED** [review]

**Review by OleksandrZhytnyk — DISMISSED** [review]

File: examples/qr-generator-demo/qr-generator-demo.py [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 import qrcode
-
+from PIL import Image
+from PIL import ImageFont
+from PIL import ImageDraw
 
 def image_generator(csv_file_path):
     df = pd.read_csv(csv_file_path, usecols=['employee_name', 'equipment_name', 'equipment_id'])
+    x=140
+    y=10
```
Comment (aartiukh) [inline examples/qr-generator-demo/qr-generator-demo.py]:
```python
top_left_text_position = (140, 10)
```
move this definition closer to its usage, i.e. line 22

File: examples/qr-generator-demo/qr-generator-demo.py [hunk 4]
```diff
@@ -12,7 +16,10 @@ def image_generator(csv_file_path):
         qr.add_data(df_list)
         qr.make()
 
+        font = ImageFont.load_default()
```
Comment (aartiukh) [inline examples/qr-generator-demo/qr-generator-demo.py]: Remove

File: examples/qr-generator-demo/qr-generator-demo.py [hunk 5]
```diff
@@ -12,7 +16,10 @@ def image_generator(csv_file_path):
...
         img = qr.make_image()
+        draw = ImageDraw.Draw(img)
+        draw.text((x, y), df["employee_name"][i], font=font)
```
Comment (aartiukh) [inline examples/qr-generator-demo/qr-generator-demo.py]:
```suggestion
        draw.text((x, y), df["employee_name"][i], font=ImageFont.load_default())
```

**Review by aartiukh — CHANGES_REQUESTED** [review]

File: examples/qr-generator-demo/qr-generator-demo.py (hunk 4 above)
Comment (MykytaKyt) [reply examples/qr-generator-demo/qr-generator-demo.py]: Removed

File: examples/qr-generator-demo/qr-generator-demo.py (hunk 5 above)
Comment (MykytaKyt) [reply examples/qr-generator-demo/qr-generator-demo.py]: Done

File: examples/qr-generator-demo/qr-generator-demo.py (hunk 3 above)
Comment (aartiukh) [reply examples/qr-generator-demo/qr-generator-demo.py]: @NikitaKit1998  ^^

**Review by aartiukh — APPROVED** [review]

**Review by paulzcooper — APPROVED** [review]

**Pull request merged**